app/ui-react/packages/ui/src/Data/DvConnection/DvConnectionCard.tsx

```diff
@@ -47,7 +47,15 @@ export const DvConnectionCard: React.FunctionComponent<
       onClick={doToggleSelected(props.name)}
     >
       <CardHeader>
-        {props.loading ? <Spinner loading={true} inline={true} /> : <></>}
```

Comment:
your change would work fine, however I think something like this would be more readable
`{ (props.loading && props.dvStatus !== ConnectionStatus.ACTIVE) ? <Spinner loading={true} inline={true} /> : <></>}`
wdyt?

Reply:
yes, it can be easy to grasp at first glance.

Reply:
updated same in PR